lib/take_away.rb

```diff
@@ -0,0 +1,22 @@
+require_relative 'order'
+require_relative 'menu'
+
+class TakeAway
+
+  def initialize(order = Order.new, menu = Menu.new)
+    @order = order
+    @menu = menu
+  end
+
+  def show_menu
+    @menu.print_menu
+  end
+
+  def order_dish(dish, qty)
+    @order.add_dish(dish, qty)
+  end
+
+  def receipt
+    @order.print_order(@menu)
+  end
+end
```

Comment:
I like the design of this class a lot. It is very clear how you made it the "runner" class, injecting the other two classes into it. It is clear throughout what you are doing here.